src/playground/utils/codemirror-linter-extension.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable no-use-before-define, no-param-reassign */
+/* eslint-disable no-use-before-define, no-param-reassign, eslint-comments/disable-enable-pair, eslint-comments/require-description -- Code follows 3rd party styles */
```

Comment:
```suggestion
/* eslint no-use-before-define: 0, no-param-reassign: 0, eslint-comments/require-description: 0 -- Code follows 3rd party styles */
```

It might be better to just turn off those rules.